lib/Sequence.fz

```diff
@@ -927,7 +927,9 @@ public Sequence(public T type) ref : property.equatable is
       else if b.finite=trit.yes
         b.count = (a.take (b.count+1)).count && equality0 a b
       else
-        equality0 a b
+        equality0 a b &&
```

Comment:
Why not sth. like `equality0 a b && a.length = b.length`?

Reply:
I see, I thought `equality0 a b` would not terminate anyway but it seems it does.